src/main/resources/latest-spark-swagger-output.yaml

```diff
@@ -67,12 +67,24 @@ paths:
       description: "Gets a list of all 'AdminUser' entities."
       produces:
       - "application/json"
-      parameters: []
+      parameters:
+      - name: "limit"
+        in: "query"
+        description: "If specified, the maximum number of items to return."
+        required: false
+        type: "string"
+        default: "10"
+      - name: "offset"
+        in: "query"
+        description: "If specified, the number of records to skip/offset."
+        required: false
+        type: "string"
+        default: "0"
       responses:
         "200":
           description: "successful operation"
           schema:
-            $ref: "#/definitions/AdminUser[]"
+            $ref: "#/definitions/ResponseList"
```

Comment:
I suppose this is a little better than the AdminUser[] type since the definition for that was just an empty object. It might be out of scope for this PR, but it would be nice to have the swagger docs include refs to the proper object in an array.

Reply:
@evansiroky Note that `AdminUser` is a subclass of `AbstractUser`, but spark-swagger ignores inherited fields at the time this was implemented. We insert inherited fields into the `AdminUser`swagger type using the `PublicApiDocGenerator` class.